models/person/person.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/local/bin/python3
+class Person(object):
+    """The person class."""
+
+    id = 0
+
+    def __init__(self, first_name, last_name, wants_accomodation="N"):
+        """Constructor function for person class."""
+        type(self).id += 1
+        self.first_name = first_name.upper()
+        self.last_name = last_name.upper()
+        self.full_name = first_name + " " + last_name
```

Comment:
there may not be need to have this variable since you can build it "on the fly" with the first_name and last_name